### PR TITLE
Bump Rails to 5.2.1

### PR DESCRIPTION
**Pull request by greysteil**
Rails 5.2.1 was released a couple of days ago!

Of the 168 updates that Dependabot has done for it it's been a straightforward upgrade [94% of the time](https://dependabot.com/compatibility-score.html?dependency-name=rails&package-manager=bundler&previous-version=5.2.0&new-version=5.2.1), so hopefully there's no work required here.🤞

### CODE REVIEW

**Comment by SamSaffron** [comment]
I support this upgrade but we are going to have to take this ourselves and carefully test it.

**Comment by tgxworld** [comment]
> it it's been a straightforward upgrade 94% of the time, so hopefully there's no work required here.

Unfortunately, we're in the 6% here :( 

This PR can only be merged after https://github.com/rails/rails/pull/33673

File: Gemfile [hunk 1]
```diff
@@ -13,13 +13,13 @@ if rails_master?
...
   gem 'seed-fu', git: 'https://github.com/SamSaffron/seed-fu.git', branch: 'discourse'
 else
-  gem 'actionmailer', '5.2'
-  gem 'actionpack', '5.2'
-  gem 'actionview', '5.2'
-  gem 'activemodel', '5.2'
-  gem 'activerecord', '5.2'
-  gem 'activesupport', '5.2'
-  gem 'railties', '5.2'
+  gem 'actionmailer', '5.2.1'
```
Comment (tgxworld) [inline Gemfile]: Let's update this to use `~> 5.2` instead.

**Comment by greysteil** [comment]
Oh nice, thanks for submitting that PR. Do you want to run against a commit SHA once it's merged or wait until 5.2.2?

**Comment by tgxworld** [comment]
@greysteil We'll be waiting for 5.2.2 to be released.

**Comment by tgxworld** [comment]
PR has been merged but we'll have to wait till Rails 5.2.2. Thank you for your contributions @greysteil  :+1:

**Pull request closed without merging**